assets/helpers/internationalisation/country.js

```diff
@@ -424,7 +421,8 @@ function handleCountryForCountryGroup(
     return null;
   }
 
-  const candidateCountry: ?IsoCountry = fromQueryParameter() || fromCookie() || fromGeolocation();
+  const candidateCountry: ?IsoCountry =
+    fromQueryParameter() || fromCookie() || fromGeolocation() || fromPath();
```

Comment:
Is this precedence correct? It goes against HTTP REST principles, however I can understand a UX reason why we'd want the path to be passive to geo location. Please can we discuss? Thanks

Reply:
I should not modify this, I roll it back in 95ce0b7ea972e6714152f0693d4ef3b41caea2b9 Nevertheless, lets catch up about this.